quick_start/utils.py

```diff
@@ -207,9 +218,10 @@ def ingest_from_file(
             :param table_name: Table name
             :param file_path: File path
             :param data_format: Given data format
+            :param ignore_first_record: Flag noting whether to ignore the first record in the table
```

Comment:
It should be made clear this is relevant for tabular textual formats (CSV and the likes).
You can also point to the below documentation
https://docs.microsoft.com/en-us/azure/data-explorer/ingestion-properties